stdlib/REPL/src/REPL.jl

```diff
@@ -1239,4 +1242,20 @@ function start_repl_server(port::Int)
     end
 end
 
+print_exit_hint(repl::BasicREPL, line) = print_exit_hint(repl.terminal, line)
+print_exit_hint(repl::LineEditREPL, line) = print_exit_hint(repl.t, line)
+print_exit_hint(repl::StreamREPL, line) = print_exit_hint(repl.stream, line)
+function print_exit_hint(io::IO, line)
+    if lowercase(strip(line)) == "exit"
+        if Sys.iswindows()
+            msg = "To exit Julia, or type exit() and press enter"
```

Comment:
Ctrl-D does seem to work on Windows, at least on the VM I tried?

Reply:
Cool. Let's just print it unconditionally.

Reply:
Well.... actually, is there a chance your VM software is remapping the keys for you?

Reply:
Several people on Slack also say that Control-D works to exit the Julia REPL on Windows.

Reply:
Alright, we now mention Control-D unconditionally.

Reply:
Good to check, thanks!